src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -114,6 +108,7 @@
       <_Dependency Remove="StreamJsonRpc"/>
       <_Dependency Remove="Nerdbank.Streams"/>
       <_Dependency Remove="System.IO.Pipelines"/>
```

Comment:
Did we need to remove the dependencies as the removed comment suggested?
```
<_Dependency Remove="Nerdbank.Streams"/>	
<_Dependency Remove="System.IO.Pipelines"/>
```

Reply:
No, actually need to keep these two.